modules/client-configuration/pages/autoinst-reinstall.adoc

```diff
@@ -0,0 +1,18 @@
+[[reinstalling-registered-systems]]
+= Reinstalling Registered Systems
+
+The in-place reinstallation is a purely local mechanism. There is therefore no need for PXE booting capabilities on the client.
+
+To reinstall a registered client in-place, you must first define an autoinstallation distribution and an autoinstallation profile.
+For details, see xref:client-configuration:autoinst-distributions.adoc[Autoinstallable Distributions] and xref:client-configuration:autoinst-profiles.adoc[Autoinstallation Profiles].
```

Comment:
```suggestion
For more information, see xref:client-configuration:autoinst-distributions.adoc[] and xref:client-configuration:autoinst-profiles.adoc[].
```

Reply:
ok, but the "**more** information" is a bit wrong, because we did not provide information yet. "More information" would suggest a complement, but there's nothing to complement.

Reply:
It's the standard formula: https://github.com/uyuni-project/uyuni-docs/wiki/wug#for-see